samples/snippets/translate_v3beta1_translate_document.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3beta1_translate_document]
+from google.cloud import translate_v3beta1 as translate
+
+
+def translate_document(project_id: str, file_path: str):
+    '''Translating a Document'''
```

Comment:
See comment above with regards to style of multi-line comments. Also, I don't think you need to repeat the name of the function here.